src/main/resources/config.sample.properties

```diff
@@ -1,4 +1,9 @@
 # Database
+#valid values are the name of the enums from: http://www.jooq.org/javadoc/3.7.x/org/jooq/SQLDialect.html
+database.dialect=MYSQL
+#configuration needed if no connection pool is provided
 database.url = jdbc:mysql://localhost:3306/crowdcontrol
 database.username = crowdcontrol
 database.password = ...
+#or if a connection pool is provided
```

Comment:
then he should simply use database.{url,username,password} 

A link can get invalid, if you want to add docs for pools add them here in the wiki and add this link.